index.d.ts

```diff
@@ -0,0 +1,25 @@
+/// <reference types="node"/>
+
+/**
+ * Read a chunk from a file asyncronously.
+ *
+ * @param filePath - The path to the file.
+ * @param startingPosition - Position to start reading.
+ * @param length - Number of bytes to read.
+ * @returns A `Promise<Buffer>` for the read chunk.
```

Comment:
I don't think we should include types in the description. The types are already in the definition.